src/components/AI/CustomPersonasSlide.tsx

```diff
@@ -0,0 +1,139 @@
+import Link from 'components/Link'
+import { graphql, useStaticQuery } from 'gatsby'
+import OSTabs from "components/OSTabs"
+import CloudinaryImage from 'components/CloudinaryImage'
+
+const content = {
+  'founders': {
+    role: 'Founders',
+    image: 'https://res.cloudinary.com/dmukukwp6/image/upload/hog_founder_adaf6ef8c1.png',
+    title: 'Move fast and break (fewer) things',
+    description: 'Ask PostHog AI to check session replays and error logs, before you waste a sprint fixing the wrong feature.',
+    features: [
+      {
+        title: 'Get a technical co-founder that never sleeps',
+        description: "Can't wait two business days for analytics? PostHog AI answers your questions instantly so you can make the call.",
+      },
+      {
+        title: "Know your growth loop before the board meeting",
+        description: "PostHog AI analyzes user behavior patterns, surfaces the ‘aha’ moment, and hands you the playbook to ‘wow’ investors.",
+      }
+    ]
+  },
+  'product-engineers': {
+    role: 'Product Engineers',
+    image: 'https://res.cloudinary.com/dmukukwp6/image/upload/hog_engineer_0eebaf7af1.png',
+    title: 'Ship faster without the context-switching tax',
+    description: 'You know how to build. PostHog AI removes the friction between idea and execution.',
+    features: [
+      {
+        title: "See outputs you can investigate",
+        description: "See the exact queries being run. Click through to debug when the AI gets it wrong (and it will sometimes).Every decision links to something you can verify.",
+      },
+      {
+        title: "Context that follows you everywhere",
+        description: "You're debugging a session replay. Ask \"Are other users hitting this error ?\". PostHog AI knows which session you're looking at.",
+      }
+    ]
+  },
+  'product-managers': {
+    role: 'Product Managers',
+    image: 'https://res.cloudinary.com/dmukukwp6/image/upload/hog_pm_23749f3c26.png',
+    title: 'Stop writing docs that nobody reads',
+    description: "Skip the PRD ceremony. Ask PostHog AI what's actually happening, prototype a fix, ship it before standup.",
+    features: [
+      {
+        title: "Run your own experiments",
+        description: "Set up A/B tests, analyze results, iterate by asking in natural language. PostHog AI puts the full power of PostHog at your fingertips",
+      },
+      {
+        title: "10x your investigation speed",
+        description: "Ask for the data that matters to you. “why did signups drop”, “is NPS trending positive?”, “show me onboarding retention by acquisition channel”.",
+      }
+    ]
+  },
+  'growth-marketers': {
+    role: 'Growth / Marketers',
+    image: 'https://res.cloudinary.com/dmukukwp6/image/upload/hog_growth_5163e429b4.png',
+    title: 'More leverage than you’ve ever had before',
+    description: "You know what questions to ask. You shouldn't need an engineering degree to get answers.",
+    features: [
```

Comment:
"More leverage than you’ve ever had before" leans marketing-y.

Maybe something more about how less-technical (don't _have_ to say that part) people are more empowered to get quantitative data to make arguments that engineers and PMs will buy into.